Add deadlocks support for Azure DB (#27549)
DataDog/documentation: content/en/database_monitoring/guide/sql_deadlock.md

: SQL Server

Supported deployments

> : All deployment types. (Azure DB with the Agent versions 7.64.0+.)

Reviewer: Is this correct?
```suggestion
: All deployment types. Azure DB is supported for Agent versions 7.64.0 and later.
```
